### ISSUE TITLE
Code documentation

### ISSUE THREAD

**Issue by MaximumViciousDeer** [inconspicuous-creations/Inconspicuous.Framework#4]
I can't find any documentation in the framework code and it would simplify learning Inconspicuous a lot if the code had [documentation comments](https://msdn.microsoft.com/en-us/library/5ast78ax.aspx).

**Comment by drony** [comment]
i agree great framework but need documention. i think is better then strange ioc. but need lot of sample
